### ISSUE TITLE
[Amarite Incompatibility] - Crash when holding a photograph [1.19.2, Quilt]

### ISSUE THREAD

**Issue by Aeonsteel** [mortuusars/Exposure#45]
Might also affect fabric.
[crash-2024-02-06_23.09.50-client.txt](https://github.com/mortuusars/Exposure/files/14189337/crash-2024-02-06_23.09.50-client.txt)

I'm on a server with a custom modpack, if that affects anything.

**Comment by Aeonsteel** [comment]
Happens when scrolling items/changing slots to a Photograph

**Comment by Aeonsteel** [comment]
[crash-2024-02-07_13.14.20-client.txt](https://github.com/mortuusars/Exposure/files/14200540/crash-2024-02-07_13.14.20-client.txt)
Previous crash report gave a false positive on the issue

**Comment by mortuusars** [comment]
This is a known conflict with Amarite. And since their mod is not open source I don't know what they do to make it not work as it should.

I'm not seeing anyting wrong on my part: 
```java
if (stack.getItem() instanceof PhotographItem || stack.getItem() instanceof StackedPhotographsItem) {
            if (isMainHand && this.offHandItem.isEmpty()) {
                exposure$renderTwoHandedPhotograph(player, poseStack, buffer, combinedLight, pitch, equipProgress, swingProgress);
            } else {
                exposure$renderOneHandedPhotograph(player, poseStack, buffer, combinedLight, equipProgress, arm, swingProgress, stack);
            }

            poseStack.popPose();

            ci.cancel();
        }
```
Item Stack is ensured to be a photograph (or a stack of photographs) before rendering. How they make it be air (empty) after that is beyond me.

**Comment by mortuusars** [comment]
This seems to have been fixed in Exposure 1.4.0
